sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md

```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
+> Ewels, P.A., Peltzer, A., Fillinger, S. et al. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol 38, 276–278 (2020). https://doi.org/10.1038/s41587-020-0439-x
 
-Other examples: ([nf-core/eager](https://peerj.com/articles/10947/#acknowledgements)).
+### Minimum Requirement: Acknowledgments Section
 
-**Optionally**, the nf-core community can be included as a consortium co-author of the publication ([example](https://doi.org/10.3390/ijms232314512)). This route is a good idea when a pipeline has used extensive existing nf-core pipeline infrastructure (e.g. modules) that were written by other members of the community not directly involved in the pipeline itself.
+At a minimum, the **nf-core** community should be acknowledged in the **Acknowledgments** section of your paper. Here’s an example from the [nf-core/clipseq preprint](https://doi.org/10.12688/wellcomeopenres.19453.1):
 
-If any members of the nf-core community have provided significant input to the creation of the pipeline, please consider adding them as coauthors on the paper directly.
+> "We would like to thank the nf-core team, particularly Harshil Patel, Phil Ewels, and Maxime Garcia, for welcoming us to the nf-core community and their invaluable help in reviewing and releasing the pipeline. We also thank the broader nf-core community for creating and maintaining the infrastructure and resources for Nextflow pipelines. A full list of nf-core community members is available at https://nf-co.re/community."
 
-If in doubt, contact the core team for guidance.
+For more examples, see the acknowledgments section of publications like [nf-core/eager](https://peerj.com/articles/10947/#acknowledgements).
+
+List of Publications about nf-core and nf-core pipelines can be found [here](https://nf-co.re/publications).
+
+### Optional: Consortium Co-authorship
+
+If your work has made extensive use of **nf-core** pipeline infrastructure (e.g., multiple modules written by other community members), consider including the **nf-core community** as a **consortium co-author**. This approach is ideal when a significant portion of the pipeline was developed by contributors who were not directly involved in your specific project. See an example of this in [this publication](https://doi.org/10.3390/ijms232314512).
+
+### Co-authorship for Significant Contributions
+
+If individual members of the **nf-core** community have made significant contributions to your pipeline’s development, please consider inviting them as co-authors on the paper. A good rule of thumb is to include those who have provided substantial input beyond the standard community support.
+
+### Need Guidance?
+
+If you’re unsure how to acknowledge **nf-core** in your publication, feel free to reach out to the **core team** for advice. They’ll be happy to guide you on the best practices for recognition.
```

Comment:
Would be good to say how to do this. I usually write "ask on slack" as that's more actionable. Could even be "Ping `@core-team` on Slack".